packages/lokse/src/commands/update.ts

```diff
@@ -43,6 +43,12 @@ class Update extends Base {
       options: outputFormats,
       description: `output format. Default is ${defaultFormat}.`,
     }),
+    sheets: flags.string({
+      char: "s",
+      name: "sheets",
+      description:
+        "sheets to get translations from. Name or list of names, comma sepaarated. For example ",
```

Comment:
sepaarated

Reply:
"Described sheets to get translations from. Name or a list of names, comma separated. For example ['Translations 1', 'Translations 2']"